ui/src/assets/i18n/de.json

```diff
@@ -0,0 +1,106 @@
+    "cancel": "Abbrechen",
+    "loading": "Wird geladen",
+    "listedBelow": "nachfolgend aufgeführten",
+    "areYouSure": "Bist du sicher?",
+    "setDefault": "Standard setzen",
+    "makeDefault": "Machen Standard"
+  },
+  "appRoot": {
+      "welcomeTo": "Willkommen zu",
+      "appTitle": "Data Flow UI",
+      "youHaveToLogin": "Sie müssen sich anmelden, um auf die Anwendung zuzugreifen.",
+      "title": "Data Flow"
+  },
+  "layout": {
+    "logo": {
+      "title": "Spring Cloud Dataflow UI"
+    },
+    "nav": {
+      "applications": "Anwendungen",
+      "streams": "Strömt",
+      "runtime": "Laufzeit",
+      "tasks": "Aufgaben",
+      "tasksJobs": "Aufgaben / Arbeitsplätze",
+      "tasksExecution": "Aufgabenausführungen",
+      "jobsExecution": "Jobausführungen",
+      "schedules": "Zeitpläne",
+      "manage": "Verwalten",
+      "auditRecords": "Audit-Aufzeichnungen",
+      "tools": "Werkzeug"
+    }
+  },
+  "applications": {
+    "main": {
+      "title": "Anwendungen",
+      "addApplications": "Anwendung(en) hinzufügen",
+      "unregisterApplications": "Anmeldung(en) abmelden",
+      "groupActions": "Gruppenaktionen",
+      "name": "Name",
+      "type": "Typ",
+      "version": "Ausführung",
+      "uri": "URI",
+      "metadataURI": "Metadaten URI",
+      "manageVersion": "Version verwalten",
+      "unregister": "Abmelden",
+      "noResult": "Keine Ergebnisse gefunden",
+      "applicationsPerPage": "AnwendungenProSeite",
+      "application": "anwendung"
+    },
+    "add": {
+      "registerOneOrMore": "Registrieren Sie eine oder mehrere Anwendungen",
+      "importFromHttp": "Importieren von Anwendungskoordinaten von einem HTTP-URI-Speicherort",
+      "importFromFile": "Importieren von Anwendungskoordinaten aus einer Eigenschaftendatei",
+      "importFromSpringDataFlow": "Importieren Sie Anwendungsstarter von dataflow.spring.io",
+      "websiteStarters": {
+        "usePredefined": "Verwenden Sie vordefinierte Anwendungs-URIs (entweder für Maven oder Docker) für alle sofort einsatzbereiten Stream- und Aufgaben-/Batch-Apps",
+        "startersForKafkaMaven": "Stream-Anwendungsstarter für Kafka/Maven",
+        "startersForKafkaDocker": "Stream-Anwendungsstarter für Kafka/Docker",
+        "startersForRabbitMQMaven": "Stream-Anwendungsstarter für RabbitMQ/Maven",
+        "startersForRabbitMQDocker": "Stream-Anwendungsstarter für RabbitMQ/Docker",
+        "startersForMaven": "Task-Anwendungsstarter für Maven",
+        "startersForDocker": "Task-Anwendungsstarter für Docker",
+        "forceImport": "Erzwingen, die Anwendungen werden importiert und installiert, auch wenn sie bereits vorhanden sind, jedoch nur, wenn sie noch nicht verwendet werden",
+        "importApplications": "Anwendung(en) importieren",
+        "importingApplications": "Anwendung(en) werden importiert..."
+      }
+    },
+    "unregister": {
+      "confirmUnregisterApplication": "Anmeldung abmelden bestätigen",
```

Comment:
Just a small hint Application(s) translates to Anwendung(en) - Anmeldung translates to Login.